src/main/scala/com/cognite/sdk/scala/v1/files.scala

```diff
@@ -94,15 +101,19 @@ final case class FileUpdate(
     sourceCreatedTime: Option[Setter[Instant]] = None,
     sourceModifiedTime: Option[Setter[Instant]] = None,
     securityCategories: Option[NonNullableSetter[Seq[Long]]] = None,
-    dataSetId: Option[Setter[Long]] = None
+    dataSetId: Option[Setter[Long]] = None,
+    labels: Option[LabelsOnUpdate] = None
```

Comment:
didn't we remove this type and instead started using:
```
final case class UpdateArray[+T](add: Seq[T] = Seq.empty, remove: Seq[T] = Seq.empty)
    extends NonNullableSetter[Seq[T]]```

Reply:
Sorry, I first implemented it in an old branch and then did a bad merge... :facepalm: Should be fixed now.